report/Sprint_2/testPiler.tex

```diff
@@ -7,9 +7,12 @@ \section{Test Generator}
 It can be compiled as a stand-alone program for any x86 machine, or it can be transpiled into Javascript, or as is the case with Flutter, it can be compiled to run on Mobile devices. 
 So choosing Dart gives us a lot of flexibility in the future if needed.
 The disadvantages with using dart is that it is a fairly new language that is not widely adopted.
-This can mean that newcomers can have a hard time understanding how it works and get qualitative learning resources.
+
+This can mean that newcomers can have a hard time understanding how it works and get qualitive learning resources.
 It also means that the language itself is not as mature as Java or C\#.
-A danger is that Dart is a language developed by a private company, meaning if the company decides to stop the development of the language or decides to remove the language completely, we would be in trouble.
+While the Dart Language is open source, and we see this as a strength, it also comes with some draw backs. 
+Due to the inmatureness of Dart there has been situation where Linux users where unable to compile Dart code, due to a change in the Linux Kernel. 
+Since Dart is Open Source and there is thereby no company behind with the incentiment to fix such issues, it took the community of Dart over a month to fix it.
```

Comment:
```suggestion
Since Dart is Open Source and thereby there is no company behind with the incentiment to fix such issues, it took the community of Dart over a month to fix it.
```